radixdlt/babylon-wallet-ios
# [ABW-2705] Make manifest text copyable

Jira ticket: ABW-2705

## Description
This makes it possible to copy the raw manifest text by longpressing, using the native control

## How to test
- Show a raw manifest
- Longpress
- Select Copy
Now the manifest text should be available for pasting

## Screenshot
![image](https://github.com/radixdlt/babylon-wallet-ios/assets/123396602/17999c7e-1be2-467e-a5ff-e246aefa0d07)

## Video (native functionality, before the button was added)

https://github.com/radixdlt/babylon-wallet-ios/assets/123396602/c027a21a-7acf-4d27-93a9-753319466d22

## PR submission checklist
